src/software/ai/navigator/path_planner/theta_star_path_planner_test.cpp

```diff
@@ -5,20 +5,16 @@
 #include "shared/constants.h"
 #include "software/geom/point.h"
 #include "software/test_util/test_util.h"
+#include "software/world/field.h"
 
-/**
- * Prints out the path formed by the points given
- * @param path_points
- */
-void printPath(std::vector<Point> path_points)
+std::vector<Point> convertPathToPoints(Path path)
```

Comment:
Could we just add a `getKnots` function to the spline class instead? (for reference, search `knots` here: https://en.wikipedia.org/wiki/Spline_(mathematics) )

Reply:
done